build.sbt

```diff
@@ -305,6 +305,13 @@ def mimaSettings(since: String = version150): Seq[Setting[_]] = {
       exclude[IncompatibleSignatureProblem]("*lagom.scaladsl.api.ServiceAcl.apply$default$1"),
       exclude[IncompatibleSignatureProblem]("*lagom.scaladsl.api.ServiceAcl.method"),
       exclude[IncompatibleSignatureProblem]("*lagom.scaladsl.api.transport.RequestHeader#RequestHeaderImpl*"),
+      // Signature problems from 2.13.0 (Factory as an alias for CanBuildFrom..)
+      exclude[IncompatibleSignatureProblem](
+        "*lagom.scaladsl.api.deser.PathParamSerializer.traversablePathParamSerializer"
+      ),
+      exclude[IncompatibleSignatureProblem](
+        "*lagom.scaladsl.api.deser.LowPriorityPathParamSerializers.traversablePathParamSerializer"
+      ),
```

Comment:
That's a problem. This means that we breaking binary compatibility in a minor version.

Reply:
This is where source inclusion could help, but I agree with you about needlessly complicating the build (which would be required to avoid this as you mentioned above).